web/concrete/bootstrap/helpers.php

```diff
@@ -263,4 +263,13 @@ function array_to_object($o, $array)
 function var_dump_safe($o, $maxDepth = true)
 {
     return Doctrine\Common\Util\Debug::dump($o, $maxDepth);
+}
+
+/**
+ * @param mixed $var
+ * @return boolean
+ */
+function is_foreachable($var)
```

Comment:
Yes sure, the most important is the fix.
the global function is not really important.